src/styles/type-scale/index.md

```diff
@@ -0,0 +1,476 @@
+      },
+      {
+        text: "25px",
+        format: "numeric"
+      }
+    ],
+    [
+      {
+        text: "16"
+      },
+      {
+        html: "<code>govuk-body-s<code>"
+      },
+      {
+        text: "16px",
+        format: "numeric"
+      },
+      {
+        text: "20px",
+        format: "numeric"
+      }
+    ]
+  ]
+}) }}
+
+{{ govukTable({
+  caption: "Small screens",
+  captionClasses: "govuk-table__caption--m",
+  head: [
+    {
+      text: "Point on type scale"
+    },
+    {
+      text: "Used by"
+    },
+    {
+      text: "Font size",
+      format: "numeric"
+    },
+    {
+      text: "Line height",
+      format: "numeric"
+    }
+  ],
+  rows: [
+    [
+      {
+        text: "80"
+      },
+      {
+        text: "Only used in exceptional circumstances"
+      },
+      {
+        text: "53px",
+        format: "numeric"
+      },
+      {
+        text: "55px",
+        format: "numeric"
+      }
+    ],
+    [
+      {
+        text: "48"
+      },
+      {
+        html: "<code>govuk-heading-xl<code>"
+      },
+      {
+        text: "32px",
+        format: "numeric"
+      },
+      {
+        text: "35px",
+        format: "numeric"
+      }
+    ],
+    [
+      {
+        text: "36"
+      },
+      {
+        html: "<code>govuk-heading-l<code>"
+      },
+      {
+        text: "27px",
+        format: "numeric"
+      },
+      {
+        text: "30px",
+        format: "numeric"
+      }
+    ],
+    [
+      {
+        text: "27"
+      },
+      {
+        text: "Only used in exceptional circumstances"
+      },
+      {
+        text: "21px",
+        format: "numeric"
+      },
+      {
+        text: "25px",
+        format: "numeric"
+      }
+    ],
+    [
+      {
+        text: "24"
+      },
+      {
+        html: "<code>govuk-heading-m</code>, <code>govuk-body-l</code>"
+      },
+      {
+        text: "21px",
+        format: "numeric"
+      },
+      {
+        text: "25px",
+        format: "numeric"
+      }
+    ],
+    [
+      {
+        text: "19"
+      },
+      {
+        html: "<code>govuk-heading-s</code>, <code>govuk-body</code>"
+      },
+      {
+        text: "19px",
+        format: "numeric"
+      },
+      {
+        text: "25px",
+        format: "numeric"
+      }
+    ],
+    [
+      {
+        text: "16"
+      },
+      {
+        html: "<code>govuk-body-s<code>"
+      },
+      {
+        text: "16px",
+        format: "numeric"
+      },
+      {
+        text: "20px",
+        format: "numeric"
+      }
+    ]
+  ]
+}) }}
+
+### Updates to the type scale
+
+In 2024, we updated the type scale to make text easier to read on smaller screens. We published this change in v5.x of GOV.UK Frontend.
+
+<details class="govuk-details">
+  <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+      View the old typography scale
+    </span>
+  </summary>
+  <div class="govuk-details__text">
+
+    {{ govukTable({
```

Comment:
2 things:
- We need another table to communicate responsive behaviour like we have above
- We need 14 on here to show that it existed on the old type scale (should we show this?)